src/Steps.js

```diff
@@ -0,0 +1,149 @@
+            formWrapper.appendChild(input1)
+
+            input2 = document.createElement('textarea')
+            input2.placeholder = 'Treść wiadomości'
+            formWrapper.appendChild(input2)
+        }
+        else if (this.type === 'kandydat') {
+            // console.log('kandydat')
+            input1 = document.createElement('input')
+            input1.placeholder = 'Imię i nazwisko'
+            formWrapper.appendChild(input1)
+
+            input2 = document.createElement('textarea')
+            input2.placeholder = 'Treść wiadomości'
+            formWrapper.appendChild(input2)
+
+            fetch('http://pliki.rekiny.wroclaw.pl/rekrutacja/stanowiska.json')
+                .then(response => response.json())
+                .then(body => {
+                    console.log(body)
+                    input3 = document.createElement('select')
+                    for (var i = 0; i < body.length; i++) {
+                        const option = document.createElement('option')
+                        option.value = body[i]
+                        option.innerText = body[i]
+                        input3.appendChild(option)
+                    }
+                    formWrapper.appendChild(input3)
+                })
+        }
+
+        const errorWrapper = document.createElement('div')
+
+        const button = document.createElement('button')
+        button.innerText = 'Wyślij'
+
+        const button2 = document.createElement('button')
+        button2.innerText = 'Powrót'
+
+        formularz.wrapperHTMLElement.innerHTML = ''
+        formularz.wrapperHTMLElement.appendChild(title)
+        formularz.wrapperHTMLElement.appendChild(stepTitle)
+        formularz.wrapperHTMLElement.appendChild(formWrapper)
+        formularz.wrapperHTMLElement.appendChild(errorWrapper)
+        formularz.wrapperHTMLElement.appendChild(button)
+        formularz.wrapperHTMLElement.appendChild(button2)
+
+        button.addEventListener('click', () => {
+            if (!input1.value || !input2.value) {
+                errorWrapper.textContent = "Któreś z pól wymaganych nie zostało uzupełnione1"
+                return
+            }
+            let body = {}
+
+            if (this.type === 'klient') {
+                body.companyName = input1.value
+                body.content = input2.value
+            }
+            else if (this.type === 'kandydat') {
+                body.name = input1.value
+                body.content = input2.value
+                body.stanowisko = input3.value
+            }
+
+            fetch('http://pliki.rekiny.wroclaw.pl/rekrutacja/api', {
+                method: 'PUT',
+                body
+            })
+            formularz.activeStep = new LastStep()
+            formularz.render()
+        })
+        button2.addEventListener('click', () => {
+            formularz.activeStep = new FirstStep()
+            formularz.render()
+        })
+    }
+}
```

Comment:
Braki stosowania **;**  Nie jest to wymagane przez JS ale w przypadku zakończenia działania np. jakiejś funkcji, jest to zalecane, chociażby dla poprawy estetyki oraz czytelności.